src/main/java/ayds/cuckoo/movieinfo/home/view/RatingProviderImplementations.kt

```diff
@@ -0,0 +1,39 @@
+package ayds.cuckoo.movieinfo.home.view
+
+import ayds.cuckoo.movieinfo.home.model.entities.Rating
+
+
+class IMDBRatingProvider : RatingProvider{
+    override fun getRating(rating: Rating): String {
+        val ratings = StringBuilder()
+        val score = rating.value.split("/").toTypedArray()
+        ratings.append("IMDB").append(" ").append(score[0]).append("\n")
```

Comment:
\n no funciona porque se esta usando Html. Usar < br >. Ya que definieron < br > como una constante en otra clase  pueden definir esa constante para accederla, aunque como las constantes se vuelven de uso comun entre clases mejor tener un archivo Constants.
```suggestion
       class Constants {
           companion object {
               const val LINE_DOWN = <br>
           }
       }
```